### PR TITLE
chore: Django bump to <=5.0

### PR DESCRIPTION
**Pull request by xdaniel3**

### CODE REVIEW

File: example/settings/base.py [hunk 1]
```diff
@@ -114,10 +114,10 @@
...
     'django_celery_extensions',
     'security',
-    'security.backends.sql',
-    'security.backends.logging',
-    'security.backends.elasticsearch',
-    'security.backends.testing',
+    'security.backends.sql.app.SecuritySQLBackend',
+    'security.backends.logging.app.SecurityLoggingBackend',
+    'security.backends.elasticsearch.app.SecurityElasticsearchBackend',
+    'security.backends.testing.app.SecurityTestingBackend',
```
Comment (Formulka) [inline example/settings/base.py:120]: for others, `default_app_config` was deprecated since 3.2

**Review by Formulka — APPROVED** [review]

**Review by rubickcz — APPROVED** [review]

**Review by HonzaSaibic — APPROVED** [review]

File: example/requirements.txt [hunk 2]
```diff
@@ -1,11 +1,11 @@
...
+skip-django-germanium @ git+https://github.com/skip-pay/germanium@tda/chore/django_bump
 flake8==5.0.4
 coveralls==3.3.1
 django-reversion==4.0.2
 responses==0.22.0
 celery==5.2.7
 freezegun==0.3.12
 elasticsearch-dsl==7.4.0
-skip-django-pyston==2.16.4.1
+skip-django-pyston @ git+https://github.com/skip-pay/django-pyston@tda/chore/django_bump
```
Comment (Formulka) [inline example/requirements.txt]:
```suggestion
skip-django-pyston==2.17.0
```

File: example/requirements.txt [hunk 3]
```diff
@@ -1,11 +1,11 @@
-Django<=3.2
-skip-django-germanium==2.3.6.1
+Django~=4.2
+skip-django-germanium @ git+https://github.com/skip-pay/germanium@tda/chore/django_bump
```
Comment (Formulka) [inline example/requirements.txt]:
```suggestion
skip-django-germanium==2.4.0
```

File: setup.py [hunk 4]
```diff
@@ -37,13 +37,13 @@ def read(fname):
         'Topic :: Internet :: WWW/HTTP :: Site Management',
     ],
     install_requires=[
-        'django>=3.1,<4.0',
+        'django>=4.2',
         'django-ipware>=3.0.2',
         'ansi2html>=1.6.0',
-        'skip-django-chamber>=0.6.16.3',
+        'skip-django-chamber @ git+https://github.com/skip-pay/django-chamber@tda/chore/django_bump',
```
Comment (Formulka) [inline setup.py]:
```suggestion
        'skip-django-chamber>=0.7.2',
```

File: setup.py [hunk 5]
```diff
@@ -37,13 +37,13 @@ def read(fname):
...
         'skip-django-choice-enumfields>=1.1.3.2',
-        'skip-django-generic-m2m-field>=0.0.9.3',
-        'skip-django-celery-extensions>=0.0.37.1',
+        'skip-django-generic-m2m-field @ git+https://github.com/skip-pay/django-generic-m2m-field@tda/chore/django_bump',
```
Comment (Formulka) [inline setup.py]:
```suggestion
        'skip-django-generic-m2m-field>=0.1.0',
```

File: setup.py [hunk 6]
```diff
@@ -37,13 +37,13 @@ def read(fname):
...
+        'skip-django-celery-extensions @ git+https://github.com/skip-pay/django-celery-extensions@tda/chore/django_bump',
```
Comment (Formulka) [inline setup.py]:
```suggestion
        'skip-django-celery-extensions>=0.1.0',
```

**Review by Formulka — APPROVED** [review]

**Pull request merged**